xml/System.Text.Encodings.Web/TextEncoder.xml

```diff
@@ -257,8 +265,7 @@
         <param name="utf8Destination">The destination buffer to which the encoded form of <paramref name="utf8Source" /> will be written.</param>
         <param name="bytesConsumed">The number of bytes consumed from the <paramref name="utf8Source" /> buffer.</param>
         <param name="bytesWritten">The number of bytes written to the <paramref name="utf8Destination" /> buffer.</param>
-        <param name="isFinalBlock">
-          <see langword="true" /> if there is further source data that needs to be encoded; otherwise, <see langword="false" />.</param>
+        <param name="isFinalBlock"><see langword="false" /> if there is further source data that needs to be encoded; otherwise, <see langword="true" />.</param>
```

Comment:
```suggestion
        <param name="isFinalBlock"><see langword="true" /> if there is no further source data that needs to be encoded; otherwise, <see langword="false" />.</param>
```

Reply:
I suggested the same thing below, but re-worded it too. @rpetrusha - thoughts on going with that wording instead (which I prefer)?

https://github.com/dotnet/dotnet-api-docs/pull/3151#discussion_r322499875